tools/import-wraps.py

```diff
@@ -137,22 +138,20 @@ def get_provide(wrap):
     deps = [i for i in deps if i]
     return progs, deps
 
-def add_to_db(wrap, versions, releases):
-    releases.setdefault(wrap, {})
-    releases[wrap].setdefault('versions', [])
-    releases[wrap].setdefault('dependency_names', [])
-    releases[wrap].setdefault('program_names', [])
-    versions = [Version(f'{version}-{revision}') for version, revision in versions]
-    versions = sorted(versions, reverse=True)
-    versions = [v._s for v in versions]
+def add_to_db(wrap: str, versions: T.List[T.Tuple[str, int]], releases: T.Dict[str, configparser.SectionProxy]):
+    releases.setdefault(wrap, {}) # type: ignore
+    releases[wrap].setdefault('versions', []) # type: ignore
+    releases[wrap].setdefault('dependency_names', []) # type: ignore
+    releases[wrap].setdefault('program_names', []) # type: ignore
+    version_objs = sorted([Version(f'{version}-{revision}') for version, revision in versions], reverse=True)
```

Comment:
This change was made because overwriting a parameter is a no-no in the type checking world.

Reply:
It mixes two changes :p one change renames the temporary variable to not be the same as the parameter, and one change moves around code so that it is sorted() in the same line rather than on a second line.

The old version built a list comprehension, assigned it, then overwrote itself by sorting itself (and you can do `versions.sort(reverse=True)`). Just pass a generator expression directly to `sorted()` (you're still using an inline list) and call it "rewriting for readability and efficiency" (mypy does not care if you assign `version_objs = [...]; version_objs = sorted(version_objs, reverse=True)`). In a separate commit, rename the actual variable for mypy purposes.